changelog.md

```diff
@@ -5,20 +5,18 @@ Bugfix release
 - [MARATHON-7576](https://jira.mesosphere.com/browse/MARATHON-7576) Changed default of UnreachableEnabled to (0,0)
 - TaskLauncherActor doesn't wait for in-flight tasks on stop
 - [MARATHON-7765](https://jira.mesosphere.com/browse/MARATHON-7765) Update to jar Manifest file
-- Replace custom IO with Apache Commons IO
-- Added SchedulerPlugin to enable the ability to
-- Update to the docker build process
+- Added SchedulerPlugin to enable the ability to customize the rejection of offers. (see below)
 - [MARATHON-2520](https://jira.mesosphere.com/browse/MARATHON-2520) Improved logging around migration
 - [D1044](https://phabricator.mesosphere.com/D1044) EventStream implementation moved to Akka eventStream
 - [D1042](https://phabricator.mesosphere.com/D1042) Fix a problem with allocating port for IT by forbidding port reuse in the `PortAllocator`
 - [D1058](https://phabricator.mesosphere.com/D1058) Fixed ephemeral port range to avoid collision with executor
```

Comment:
I think we can leave out this test-specific change

Reply:
there were port range changes that were in prod code...but I can remove